### ISSUE TITLE
Chatbot usage for blind users

### ISSUE THREAD

**Issue by cengizmurat** [botpress/v12#940]
Hello,

**Problem description**
I have a **blind developer** in our team and he has some difficulties to use Botpress chatbot.
He works with two screen readers *JAWS* and *NVDA*, and there are different issues for interacting with Botpress chatbot : 
1. To access the chatbot's widget: the button is not labeled and the screen reader says : `not labeled button`. He is not aware of what he's opening and goes into the unknown.
2. When he sends a message to your chatbot he does not have the focus to the new message(s) for reading it/them. He has to search the chatbot's new answer(s) among all the ongoing discussion, from the very first message to the newest ones (for each message sent by him).

**Solution/Alternatives**
- A labelled widget button according the [Web Content Accessibility Guidelines](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#contents) in order for disabled users to know what this button is for
- An option for navigating easily (ex: with tab key) between user input field and only chatbot's new messages.

Thank you very much,
Best regards
